### PR TITLE
Remove kind specific code

### PR DESCRIPTION
**Pull request by filariow**
Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

**Review by mmulholla — CHANGES_REQUESTED** [review]
Code looks fine but the README needs updating with the new flag and some info on how to get the url.
My PR for dry run has quite a few doc updates and will clash, so better if we can merge that first and then update the readme.

File: README.md [hunk 1]
```diff
@@ -328,21 +331,21 @@ options:
...
                         Config file containing agent roles
   -t TENANT, --tenant TENANT
                         tenant to use. Default: primaza-system
+  -u TENANT_INTERNAL_URL, --tenant-internal-url TENANT_INTERNAL_URL
+                        Internal URL for the cluster on which Primaza's Control Plane is running
   -v VERSION, --version VERSION
                         Version of primaza to use, default: latest. Ignored if --config is set.
   -k KUBECONFIG, --kubeconfig KUBECONFIG
-                        path to kubeconfig file, default: KUBECONFIG environment variable if set, otherwise /<home directory>/.kube/config
+                        path to kubeconfig file, default: KUBECONFIG environment variable if set, otherwise /home/filario/.kube/config
```
Comment (mmulholla) [inline README.md]: remove ref to /home/filario.

File: README.md [hunk 2]
```diff
@@ -416,28 +420,24 @@ options:
...
```
Comment (mmulholla) [inline README.md]: remove ref to /home/filario

**Review by mmulholla — CHANGES_REQUESTED** [review]
two minor comments.

File: README.md (hunk 1 above)
Comment (filariow) [reply README.md]: great catch! done

**Review by mmulholla — APPROVED** [review]

**Pull request merged**